rfcs/036-holdings-records/README.md

```diff
@@ -0,0 +1,261 @@
+    These fields are used to describe which parts of a multi-part object we have: for example, the individual issues of a journal.
+
+    The fields come in pairs (853 is labels, 863 is values).
+    For example, 853 might contain labels `vol.`, `no.` and `issue`, and 863 might contain the values `1–9`, `1–90`, `2000–2010`.
+    These would be joined to form the string `vol. 1–9 no. 1–90 issue 2000–2010`.
+
+    There are similar pairs 854/864 (for supplementary material) and 855/865 (for captions and indexes).
+
+*   [**856 Electronic Location.**](https://www.oclc.org/bibformats/en/8xx/856.html)
+    See [RFC 035](https://github.com/wellcomecollection/docs/tree/master/rfcs/035-marc-856) about how we model electronic locations for more discussion of this field.
+
+*   **949 Shelfmark.**
+    This contains values like /JOU, /HIS and /MED.
+    These are currently exposed on wellcomelibrary.org, e.g. [b13488284](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1348828?lang=eng)
+
+*   **[843 Reproduction Note](https://help.oclc.org/Metadata_Services/Local_Holdings_Maintenance/OCLC_MARC_local_holdings_format_and_standards/8xx_fields/843_Reproduction_Note).**
+    It's not clear how we're actually using this field: it contains values like "Library Display" and "Reading Room", accompanied by a range of dates, sometimes years passed.
+
+We also have a handful of records using fields 000, 561, 596, 852, 860, 876, 990.
+Because these are so infrequently used, we will skip using them in the initial model.
+
+----
+
+
+
+
+
+
+
+## How should we present holdings records in the Catalogue API?
+
+### Can we present holdings records as Items?
+
+The Catalogue API already has a model for `Work` and `Item`.
+The Work model is populated from bibs, and the `Item` model from items.
+
+We did consider trying to put data from holdings records into the `Item` model, but the data in holdings records doesn't really fit.
+For example:
+
+-   Two items titled *Volume 1* and *Volume 2*, and a holdings record titled *Complete set*.
+    It's useful to know that these two volumes form a complete set, but it wouldn't make sense to store this information on the individual items.
+
+-   Two items titled *Volume 1* and *Volume 2*, and a holdings record titled *Missing volume 3*.
+    Here the holdings record is even more important, but there's no way to encode the absence of an item.
+
+-   A run of journals, with each issue itemised, and a holdings record titled *100 issues, 1957–1972*.
+    The holdings record can provide useful summary information that isn't easily observed from the items.
+
+### Creating a separate Holdings type
+
+Rather than reuse the `Item` type, we will create a new `Holdings` type with the same fields (at least, initially).
+This will appear as a list on a Work called `"holdings"`.
+
+We will populate it as follows:
+
+*   The `title` will come from the concatenated contents of MARC field 866 subfields $a and $z.
```

Comment:
I wonder if we should structure a little bit more than this, but it does depend on the data. Eg, should be have `description`, `note`, `enumeration[]`? Or is that overkill for what is usually there. I'm thinking about leaving just enough structure there so that the front end doesn't need to understand the MARC, but can choose how to render large lists.

Reply:
I went back and had another look at this; in one case there are **86** parts on a single record, so this isn't going to fly.

We aren't particularly consistent about using subfields $a and $z, but for now I've split them out as separate. They can always be combined in the front-end, and making the distinction explicit might improve the cataloguing guidelines.